databuilder/databuilder/models/atlas_serializable.py

```diff
@@ -0,0 +1,71 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+import abc
+from typing import Union  # noqa: F401
+
+from databuilder.models.atlas_entity import AtlasEntity
+from databuilder.models.atlas_relationship import AtlasRelationship
+
+GUID = 'guid'
```

Comment:
idea: https://github.com/amundsen-io/amundsen/pull/1103 introduces atlas_utils.py in amundsen_common which could be reused here